RELEASE_NOTES.md

```diff
@@ -7,51 +7,37 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# knife-azure 1.7.0 release notes:
-In this release, few options have been removed since their support has been removed from the azure-chef-extension. The options are: `--auto-update-client`, `--uninstall-chef-client` and `--delete-chef-extension-config`. They were earlier supported with `cloud-api` bootstrap protocol which uses azure-chef-extension for installing chef-client.
+# knife-azure 1.8.0 release notes:
+In this release `--chef-service-interval` option is renamed to `--chef-daemon-interval`.
+
+Updated code to work with latest azure-sdk gems i.e. version 0.9.0.
 
 New options introduced:
-`--chef-service-interval`, which lets the user specify the frequency at which chef-client service should run. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
 
-`--extended-logs` shows chef convergence logs in detail. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
+`--daemon` this allows user to select options to run chef-client as auto, service or scheduled task. This option works for windows node and --bootstrap-protocol to be 'cloud-api'.
```

Comment:
`--daemon` option accepts `none`, `service` and `task`. It would be good to clarify this as we don't pass `auto`.